### PR TITLE
Release 4.9.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-release-4.9.1](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-release-4.9.1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.9.1 (2021-06-14)
+
+- Added support for `marked@2.0.7`. [#163](https://github.com/blackbaud/skyux-docs-tools/pull/163)
```
Comment (blackbaud-paulah) [inline CHANGELOG.md:3]:
```suggestion
- Added support for `marked@2.0.7` and updated tests to be screen size agnostic. [#163](https://github.com/blackbaud/skyux-docs-tools/pull/163)
```

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md:3]: @blackbaud-paulah We don't normally call out changes to our internal tests in our CHANGELOG as it doesn't affect the consumer of the library. Thoughts?

Comment (blackbaud-paulah) [reply CHANGELOG.md:3]: Cool. I didn't know that. Thanks for explaining.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-release-4.9.1](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-release-4.9.1)

**Review by blackbaud-paulah — APPROVED** [review]

**Pull request merged**